plugins/kordamp-parentpom/src/main/groovy/org/kordamp/gradle/KordampParentPomPlugin.groovy

```diff
@@ -32,6 +32,8 @@ class KordampParentPomPlugin implements Plugin<Project> {
     void apply(Project project) {
         project.plugins.apply(JavaProjectPlugin)
 
+        if (!project.hasProperty('nexusUrl')) project.ext.nexusUrl = 'https://s01.oss.sonatype.org/service/local/'
+        if (!project.hasProperty('snapshotRepositoryUrl')) project.ext.snapshotRepositoryUrl = 'https://s01.oss.sonatype.org/content/repositories/snapshots/'
```

Comment:
Could you rename these two properties to
- sonatypeReleaseRepositoryUrl
- sonatypeSnapshotRepositoryUrl

Thanks!